addon/components/markdown-to-html.js

```diff
@@ -1,19 +1,20 @@
-import Ember from 'ember';
+import { htmlSafe } from '@ember/string';
+import Component from '@ember/component';
+import { get, computed } from '@ember/object';
+import { merge, assign } from '@ember/polyfills';
+import { getOwner } from '@ember/application';
 import showdown from 'showdown';
 import layout from '../templates/components/markdown-to-html';
 
-const { computed, get, merge, getOwner } = Ember;
 const CONFIG_LOOKUP = 'config:environment';
 
-let assign = Ember.assign;
```

Comment:
not super familiar with what is going on here, but this change is making all the tests blow up.  any ideas?

Reply:
I'll fix this, merging first so that I don't have to fork your fork

Reply:
Cool, I’ll check that out

Reply:
All fixed - looks like assign/merge were never used so was an easy fix (removed them).  I'll publish a new minor version tonight.

Reply:
Nice, I thought that might be the case but couldn’t figure out based on the blame why it was added in the first place.